addons/website_event_sale/views/website_sale_templates.xml

```diff
@@ -80,7 +80,12 @@
                                 </div>
                                 <div class="col-lg-8 p-4">
                                     <h3 t-esc="event.name" />
-                                    <a class="btn btn-primary mb-2" t-attf-href="/event/#{ slug(event) }">Go to Event</a>
+                                    <t t-set="attendee_ids" t-value="attendee_ids_per_event.get(event, [])"/>
```

Comment:
Strange does that mean you can have a flow in a given event with tickets on several events ?

Reply:
Yes but that is probably uncommon... I did not see the foreach above, hence not understanding why we kept only one event in the end. Ok then, if tickets are effectively rendered / event that's good.